torch/_dynamo/variables/lists.py

```diff
@@ -1130,6 +1156,14 @@ def call_method(
                 raise_observed_exception(AttributeError, tx)
             # Subclass of namedtuple type can have dynamic attributes
             tx.output.side_effects.mutation(self)
+            if self.source:
+                tx.output.side_effects.track_object_existing(self.source, self)
```

Comment:
I thought you had this and `self.mutation_type` below somewhere in `builder.py` and `user_defined.py` (`UserClassObjectVariable.call_function`)? That's actually the preferred approach since we'd like to set `mutation_type` as close to VariableTracker creation as possible, rather than changing them dynamically.

I probably created some confusion, I meant you can keep these modifications in those files, while removing their `is_structseq` checks, since we are already checking those in `call_method` here.